ci_framework/roles/edpm_deploy_baremetal/tasks/main.yml

```diff
@@ -0,0 +1,95 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Set install_yamls Makefile environment variables
+  vars:
+    operators_build_output: "{{ (cifmw_operator_build_output | default({'operators':{}})).operators }}"
+  ansible.builtin.set_fact:
+    cifmw_edpm_deploy_baremetal_common_env:
+      PATH: "{{ cifmw_path }}"
+    cifmw_edpm_deploy_baremetal_make_openstack_env: |
+      {% if cifmw_operator_build_meta_name is defined and cifmw_operator_build_meta_name in operators_build_output %}
+      OPENSTACK_IMG: {{ operators_build_output[cifmw_operator_build_meta_name].image_catalog }}
+      OPENSTACK_BRANCH: ""
+      GIT_CLONE_OPTS: "-l"
+      OPENSTACK_REPO: {{ operators_build_output[cifmw_operator_build_meta_name].git_src_dir }}
+      {% endif %}
+    cifmw_edpm_deploy_baremetal_operators_build_output: "{{ operators_build_output }}"
+
+- name: Prepare storage in CRC
```

Comment:
isn't that one already done in some other steps?

Reply:
Removed the crc step from the role as we i.e use STORAGE_CLASS: crc-csi-hostpath-provisioner or set via pre hooks